token/parse.go

```diff
@@ -171,7 +171,10 @@ func parseResponse(jwt *jose.JSONWebToken, p Payload) (*JSONWebToken, error) {
 		if err := json.Unmarshal(p.Amazon.Document, &p.Amazon.InstanceIdentityDocument); err != nil {
 			return nil, errors.Wrap(err, "error unmarshaling instance identity document")
 		}
-	case strings.HasPrefix(p.Issuer, "https://sts.windows.net/"):
+	case strings.HasPrefix(p.Issuer, "https://sts.windows.net/"),
+		strings.HasPrefix(p.Issuer, "https://sts.chinacloudapi.cn/"),
+		strings.HasPrefix(p.Issuer, "https://sts.microsoftazure.de/"),
+		strings.HasPrefix(p.Issuer, "https://sts.usgovcloudapi.net/"):
```

Comment:
@rvichery This looks ok, but I cannot find docs for any of those. 
Moreover, `sts.microsoftazure.de` does not resolve, `sts.cloudapi.de` does but it does not respond. And the TLS certificate for https://sts.usgovcloudapi.net is not valid for that domain.

I've asked a question here https://learn.microsoft.com/en-us/answers/questions/1190472/what-are-the-token-issuers-for-the-sovereign-cloud

Reply:
Based on the answer provided to your question at https://learn.microsoft.com/en-us/answers/questions/1190472/what-are-the-token-issuers-for-the-sovereign-cloud, I will update the above case statement.

Reply:
Thanks, I'm approving and merging.